pkg/kubectl/cmd/annotate/annotate.go

```diff
@@ -41,8 +41,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/util/templates"
 )
 
-// AnnotateOptions have the data required to perform the annotate operation
-type AnnotateOptions struct {
+// Options have the data required to perform the annotate operation
```

Comment:
Don't change these.

Reply:
Thanks for the review! I wasn't sure about these while changing them. But since we already split these packages, could you explain more about the naming preferences?

Reply:
@idealhack thanks for adding co-author information.
Just to avoid duplication of effort I would like to mention that golint fails because it prefers the name `annotate.Options` than `annotate.AnnotateOptions`.

Reply:
We don't need to blindly follow all golint suggestions, I consider this just that. That's why I think we should leave the current naming scheme as is.

Reply:
@soltysh PTAL